client/strings/en-us.json

```diff
@@ -147,6 +150,7 @@
   "HeaderNewAccount": "New Account",
   "HeaderNewLibrary": "New Library",
   "HeaderNotifications": "Notifications",
+  "HeaderOpenedRSSFeeds": "ABS Hosted RSS Feed",
```

Comment:
We probably want to update other references to RSS feeds so everything is saying "Host/Hosted" instead of "Open".

This won't affect this key since it's a new one, but if we do change the language from "Open" we'll want to make new keys and reference those keys so there aren't any translation issues. Not sure if that matters for this PR.